packages/teams-js/src/internal/communication.ts

```diff
@@ -741,6 +903,29 @@ function createMessageRequest(
   };
 }
 
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * Creates a nested app authentication request.
+ *
+ * @param {string} message - The message to be included in the request. This is typically a stringified JSON object containing the details of the authentication request.
+ * The reason for using a string is to allow complex data structures to be sent as a message while avoiding potential issues with object serialization and deserialization.
+ *
+ * @returns {NestedAppAuthRequest} Returns a NestedAppAuthRequest object with a unique id, the function name set to 'nestedAppAuthRequest', the current timestamp, an empty args array, and the provided message as data.
+ */
+function createNestedAppAuthRequest(message: string): NestedAppAuthRequest {
+  return {
+    id: CommunicationPrivate.nextMessageId++,
+    func: 'nestedAppAuthRequest',
```

Comment:
After having a discussion with the team, the func name should follow the pattern of 'capabilityName.ApiName.'. Could you please make the necessary update? 'nestedAppAuth.execute' seems good to me. Feel free to use any alternative name.

Reply:
Done.